docs/en/ingest-management/commands.asciidoc

```diff
@@ -31,31 +31,88 @@ NOTE: You might need to log in as a root user to run these commands.
 [[elastic-agent-enroll-command]]
 == elastic-agent enroll
 
-Enroll the {elastic-agent} in {fleet}.
+Enroll the {agent} in {fleet}.
+
+Use this command to enroll the {agent} in {fleet} without installing the
+agent as a service. You might want to do this if you installed
+the {agent} from a DEB or RPM package and plan to use systemd commands to
+start and manage the service. This command is also useful for testing
+{agent} prior to installing it.
 
-We recommend that you run this command as the root user because some
+TIP: To enroll an {agent} _and_ install it as a service, use the
+<<elastic-agent-install-command,`install` command>> instead.
+
+We recommend that you run the `enroll` command as the root user because some
 integrations require root privileges to collect sensitive data. This command
 overwrites the `elastic-agent.yml` file in the agent directory.
 
+This command includes optional flags to set up <<fleet-server,{fleet-server}>>.
+
+IMPORTANT: This command enrolls the {agent} in {fleet}; it does not start the
+agent. To start the agent, either <<start-elastic-agent,start the service>>, if
+one exists, or use the <<elastic-agent-run-command,`run` command>> to start the
+agent from a terminal.
+
+
 [discrete]
 === Synopsis
 
+//QUESTION: Which flags are relevant when running a Fleet
```

Comment:
Is there a way to avoid having duplicated base content here?  The enroll and 'install' commands accept the exact same options.  Their inherent difference is that 'install' additionally manages the service account and runs the Agent for the user, and while this is not required, it is recommended to avoid user error and allow the Agent to be upgradeable via the Kibana UI when subsequent stack releases are deployed.

I do think we'll want to highlight different options for when Fleet Server is included, I'll make a pass below at arranging per your notes here.

Reply:
Yah, I see your point. We could use a pipe character (like in EBNF) to show alternation, but we don't do that anywhere else AFAIK (our syntax diagrams are very simple). Could be confusing to users. I'm personally a fan of those old syntax diagrams that show all the things (like optionality, defaults, etc), but it's probably good that modern command lines don't require them. :-) 

Maybe this is something to resolve in 7.14?

Reply:
7.14 is totally fine

Reply:
Closing because I've opened https://github.com/elastic/observability-docs/issues/704 to track this.